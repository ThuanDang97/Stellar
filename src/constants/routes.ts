export const DEFAULT_HEADER_URL = {
  SIGN_UP: { TITLE: 'Sign Up', URL: '/sign-up' },
  MARKETING: { TITLE: 'Marketing', URL: '/marketing' },
  GOAL: { TITLE: 'Goal', URL: '/goal' },
  CONNECT_BANK: { TITLE: 'Connect Bank', URL: '/connect-bank' },
  LOGIN: { TITLE: 'Login', URL: '/login' },
  RESET_PASSWORD: { TITLE: 'Reset Password', URL: '/reset-password' },
  CHANGE_PASSWORD: { TITLE: 'Change Password', URL: '/change-password' },
  FORGOT_PASSWORD: { TITLE: 'Forgot Password', URL: '/forgot-password' },
  EMAIL_SENT: { TITLE: 'Email Sent', URL: '/email-sent' },
}

export const PRIMARY_HEADER_URL = {
  DASHBOARD: { TITLE: 'Dashboard', URL: '/dashboard' },
  FREE_CREDIT_SCORE: { TITLE: 'Free Credit Score', URL: '/free-credit-score' },
  CONFIRM_IDENTITY: { TITLE: 'Confirm Identity', URL: '/confirm-identity' },
  CONFIRM_ADDRESS: { TITLE: 'Confirm Address', URL: '/confirm-address' },
  CONFIRM_SSN: { TITLE: 'Confirm SSN', URL: '/confirm-ssn' },
  CONNECT_ACCOUNT: { TITLE: 'Connect Account', URL: '/connect-account' },
  CONFIRM_DETAIL: { TITLE: 'Confirm Detail', URL: '/confirm-detail' },
  LINK_FIRST_BILL: { TITLE: 'Link First Bill', URL: '/link-first-bill' },
  HELP: { TITLE: 'Help', URL: '/help' },
  FEEDBACK: { TITLE: 'Feedback', URL: '/feedback' },
  REFER_A_FRIEND: { TITLE: 'Refer A Friend', URL: '/refer-a-friend' },
  USER_PROFILE: { TITLE: 'Profile', URL: '/profile' },
  PRIVACY_POLICY: { TITLE: 'Privacy Policy', URL: '/privacy-policy' },
  TERMS_OF_USE: { TITLE: 'Terms of Use', URL: '/terms-of-use' },
  LEARN_MORE: { TITLE: 'Learn more', URL: '/learn-more' },
  ADD_FIRST_BILL: { TITLE: 'Add First Bill', URL: '/add-first-bill' },
}

export const SECONDARY_URL = {
  WELCOME: { TITLE: 'Welcome', URL: '/welcome' },
  LANDING_PAGE: { TITLE: 'Landing Page', URL: '/' },
  SUBSCRIPTION: { TITLE: 'Subscription', URL: '/subscription' },
}

export const ARROW_BACK_ROUTES = [
  DEFAULT_HEADER_URL.SIGN_UP.URL,
  DEFAULT_HEADER_URL.MARKETING.URL,
  DEFAULT_HEADER_URL.GOAL.URL,
  DEFAULT_HEADER_URL.CONNECT_BANK.URL,
]

export const PUBLIC_ROUTES = [
  DEFAULT_HEADER_URL.SIGN_UP.URL,
  DEFAULT_HEADER_URL.LOGIN.URL,
  DEFAULT_HEADER_URL.RESET_PASSWORD.URL,
  DEFAULT_HEADER_URL.CHANGE_PASSWORD.URL,
  DEFAULT_HEADER_URL.FORGOT_PASSWORD.URL,
  DEFAULT_HEADER_URL.EMAIL_SENT.URL,
  SECONDARY_URL.LANDING_PAGE.URL,
]
