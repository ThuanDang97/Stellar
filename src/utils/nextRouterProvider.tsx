import { NextRouter } from 'next/router'
import { RouterContext } from 'next/dist/shared/lib/router-context'

const createMockRouter = (router?: Partial<NextRouter>): NextRouter => ({
  basePath: '',
  pathname: '/',
  route: '/',
  query: {},
  asPath: '/',
  back: jest.fn(),
  beforePopState: jest.fn(),
  prefetch: jest.fn(),
  push: jest.fn(),
  reload: jest.fn(),
  replace: jest.fn(),
  events: {
    on: jest.fn(),
    off: jest.fn(),
    emit: jest.fn(),
  },
  isFallback: false,
  isLocaleDomain: false,
  isReady: true,
  defaultLocale: 'en',
  domainLocales: [],
  isPreview: false,
  ...router,
})

export const NextRouterProvider = ({
  children,
  router = { query: {} },
}: {
  children: React.ReactNode
  router: Partial<NextRouter>
}) => (
  <RouterContext.Provider value={createMockRouter(router)}>
    {children}
  </RouterContext.Provider>
)
