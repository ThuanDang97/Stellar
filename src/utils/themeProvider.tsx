import { ThemeProvider } from 'styled-components'
import { render } from '@testing-library/react'
import { theme } from '@themes/index'

export const renderWithTheme = (component: React.ReactNode) => {
  return render(<ThemeProvider theme={theme}>{component}</ThemeProvider>)
}
