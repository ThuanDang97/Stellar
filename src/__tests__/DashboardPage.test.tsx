import React from 'react'

// Utils
import { renderWithTheme } from '@utils/themeProvider'
import { NextRouterProvider } from '@utils/nextRouterProvider'

// Page
import DashboardPage from '@pages/dashboard'

// Constants
import { PRIMARY_HEADER_URL } from '@constants/routes'

const query = { fromPage: PRIMARY_HEADER_URL.CONFIRM_DETAIL.URL }

describe('Dashboard Page', () => {
  it('renders dashboard Page is lock unchanged', () => {
    const { container } = renderWithTheme(
      <NextRouterProvider router={{ query }}>
        <DashboardPage />
      </NextRouterProvider>,
    )

    expect(container).toMatchSnapshot()
  })
})
