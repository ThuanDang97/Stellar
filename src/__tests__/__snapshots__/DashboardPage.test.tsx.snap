// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Page renders dashboard Page is lock unchanged 1`] = `
<div>
  <div
    class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__SubBackgroundHeader-sc-2iw14z-8 lapCRK fKBABW"
    height="150"
    width="100%"
  />
  <div
    class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__WrapperMainContent-sc-2iw14z-1 jXJeDL ljMWJf"
    display="flex"
    width="100%"
  >
    <div
      class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__WrapperTableContent-sc-2iw14z-2 ksBfn erXaYX"
      display="flex"
    >
      <div
        class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__AnimatedBanner-sc-2iw14z-4 fTsvHV dnYmJu"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__BoxShadowStyled-sc-rqe5sa-2 BannerStyled-sc-587aj7-0 fTsvHV csavJt SFwKm"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 kcXXQi"
            width="55%"
          >
            <span
              class="TextStyled-sc-9xia41-0 TextStyled__TitleSecSionStyled-sc-9xia41-1 jEbBXO fYtVag"
              font-family="AdobeCleanExtraBold"
              font-size="24"
            >
              You’re Almost Done
            </span>
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2720%27%20height=%2720%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="flame"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
            <p
              class="TextStyled-sc-9xia41-0 TextStyled__TextDescriptionStyled-sc-9xia41-2 gnCdZZ fGqNzw"
              font-size="16"
            >
              Let’s set up your first bill and Stellar will make sure they get paid on time!
            </p>
          </div>
          <div
            class="BannerStyled__ImageWrapper-sc-587aj7-1 eQlXBn"
          >
            <span
              style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
            >
              <img
                alt="space-dog"
                data-nimg="fill"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
        </div>
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__WrapperPointContentMobile-sc-2iw14z-7 ifPAwR ifyfSe"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__BoxShadowStyled-sc-rqe5sa-2 djsesE dyGpHf"
          width="100%"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__AnimatedButtonUnlock-sc-2iw14z-0 jTzNtY kfEbIu"
          >
            <a
              class="LinkStyled__ButtonLinkStyled-sc-626068-1 kJrkKO"
              data-testid="link"
              href="/free-credit-score"
              width="230"
            >
              Unlock Credit Score
            </a>
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 ViewStyled__BackDropPointLocker-sc-rqe5sa-7 dMtJMR WdHgc"
            display="flex"
            width="260"
          >
            <span
              class="TextStyled-sc-9xia41-0 hRNVve"
              color="#E6E6E6"
              font-size="64"
            >
              728
            </span>
            <div
              class="ViewStyled-sc-rqe5sa-0 fYAPZY"
            >
              <div
                class="text-row"
                style="width: 5.625rem; height: 0.9375rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
              />
              <div
                class="text-row"
                style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0.625rem;"
              />
            </div>
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 cEttAk"
            display="flex"
          >
            <div
              class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__StartBaseLineStyled-sc-okjczv-1 fZhgsY ifLECW"
              height="7"
              opacity="0.2"
              width="51"
            />
            <div
              class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__MiddleBaseLineStyled-sc-okjczv-2 bgjtXS dbuCnc"
              height="7"
              opacity="0.2"
              width="16"
            />
            <div
              class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__MiddleBaseLineStyled-sc-okjczv-2 iRkeGL dbIHPc"
              height="7"
              opacity="0.2"
              width="14"
            />
            <div
              class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__EndBaseLineStyled-sc-okjczv-3 cNBCGI fKiEZo"
              height="7"
              opacity="0.2"
              width="20"
            />
            <div
              class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__LineScoreStyled-sc-okjczv-4 ipAKRB hjZOmd"
              opacity="0.7"
              width="14"
            />
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__BackDropPointLocker-sc-rqe5sa-7 fTsvHV WdHgc"
        />
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 dzPLrs"
      >
        <span
          class="TextStyled-sc-9xia41-0 bABhml"
          color="#000"
          font-size="20"
          font-weight="700"
          opacity="0.2"
        >
          Linked Bills
        </span>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__RowStyled-sc-rqe5sa-1 fTsvHV eRHuZf"
        >
          <div
            class="rect-shape"
            style="background-color: rgb(230, 230, 230); width: 2.5rem; height: 2.5rem; margin-right: 10px; margin-top: 0rem;"
          />
          <div
            class="ViewStyled-sc-rqe5sa-0 fNNXne"
          >
            <div
              class="text-row"
              style="width: 4.375rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.375rem; background-color: rgb(230, 230, 230); margin-top: 0.3125rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 bwSFDl"
          >
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 hyFUvf"
          >
            <div
              class="text-row"
              style="width: 3.125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="text-row"
            style="width: 0.8125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
          />
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__RowStyled-sc-rqe5sa-1 fTsvHV eRHuZf"
        >
          <div
            class="rect-shape"
            style="background-color: rgb(230, 230, 230); width: 2.5rem; height: 2.5rem; margin-right: 10px; margin-top: 0rem;"
          />
          <div
            class="ViewStyled-sc-rqe5sa-0 fNNXne"
          >
            <div
              class="text-row"
              style="width: 4.375rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.375rem; background-color: rgb(230, 230, 230); margin-top: 0.3125rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 bwSFDl"
          >
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 hyFUvf"
          >
            <div
              class="text-row"
              style="width: 3.125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="text-row"
            style="width: 0.8125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
          />
        </div>
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 bMAwAU"
      >
        <span
          class="TextStyled-sc-9xia41-0 bABhml"
          color="#000"
          font-size="20"
          font-weight="700"
          opacity="0.2"
        >
          Finish Adding Bills
        </span>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__RowStyled-sc-rqe5sa-1 fTsvHV eRHuZf"
        >
          <div
            class="rect-shape"
            style="background-color: rgb(230, 230, 230); width: 2.5rem; height: 2.5rem; margin-right: 10px; margin-top: 0rem;"
          />
          <div
            class="ViewStyled-sc-rqe5sa-0 fNNXne"
          >
            <div
              class="text-row"
              style="width: 4.375rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.375rem; background-color: rgb(230, 230, 230); margin-top: 0.3125rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 bwSFDl"
          >
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 hyFUvf"
          >
            <div
              class="text-row"
              style="width: 3.125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="text-row"
            style="width: 0.8125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
          />
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__RowStyled-sc-rqe5sa-1 fTsvHV eRHuZf"
        >
          <div
            class="rect-shape"
            style="background-color: rgb(230, 230, 230); width: 2.5rem; height: 2.5rem; margin-right: 10px; margin-top: 0rem;"
          />
          <div
            class="ViewStyled-sc-rqe5sa-0 fNNXne"
          >
            <div
              class="text-row"
              style="width: 4.375rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.375rem; background-color: rgb(230, 230, 230); margin-top: 0.3125rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 bwSFDl"
          >
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 hyFUvf"
          >
            <div
              class="text-row"
              style="width: 3.125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="text-row"
            style="width: 0.8125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
          />
        </div>
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 bMAwAU"
      >
        <span
          class="TextStyled-sc-9xia41-0 bABhml"
          color="#000"
          font-size="20"
          font-weight="700"
          opacity="0.2"
        >
          My Goals
        </span>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__RowStyled-sc-rqe5sa-1 fTsvHV eRHuZf"
        >
          <div
            class="rect-shape"
            style="background-color: rgb(230, 230, 230); width: 2.5rem; height: 2.5rem; margin-right: 10px; margin-top: 0rem;"
          />
          <div
            class="ViewStyled-sc-rqe5sa-0 fNNXne"
          >
            <div
              class="text-row"
              style="width: 4.375rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 bwSFDl"
          >
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 hyFUvf"
          >
            <div
              class="text-row"
              style="width: 3.125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="text-row"
            style="width: 0.8125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
          />
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__RowStyled-sc-rqe5sa-1 fTsvHV eRHuZf"
        >
          <div
            class="rect-shape"
            style="background-color: rgb(230, 230, 230); width: 2.5rem; height: 2.5rem; margin-right: 10px; margin-top: 0rem;"
          />
          <div
            class="ViewStyled-sc-rqe5sa-0 fNNXne"
          >
            <div
              class="text-row"
              style="width: 4.375rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 bwSFDl"
          >
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 hyFUvf"
          >
            <div
              class="text-row"
              style="width: 3.125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
          </div>
          <div
            class="text-row"
            style="width: 0.8125rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
          />
        </div>
      </div>
    </div>
    <aside
      class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__WrapperPointContent-sc-2iw14z-3 gmyIZz PdFpk"
      overflow="hidden"
    >
      <div
        class="ViewStyled-sc-rqe5sa-0 ViewStyled__BoxShadowStyled-sc-rqe5sa-2 djsesE dyGpHf"
        width="100%"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 DashBoardPageStyled__AnimatedButtonUnlock-sc-2iw14z-0 jTzNtY kfEbIu"
        >
          <a
            class="LinkStyled__ButtonLinkStyled-sc-626068-1 kJrkKO"
            data-testid="link"
            href="/free-credit-score"
            width="230"
          >
            Unlock Credit Score
          </a>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 ViewStyled__BackDropPointLocker-sc-rqe5sa-7 dMtJMR WdHgc"
          display="flex"
          width="260"
        >
          <span
            class="TextStyled-sc-9xia41-0 hRNVve"
            color="#E6E6E6"
            font-size="64"
          >
            728
          </span>
          <div
            class="ViewStyled-sc-rqe5sa-0 fYAPZY"
          >
            <div
              class="text-row"
              style="width: 5.625rem; height: 0.9375rem; background-color: rgb(230, 230, 230); margin-top: 0rem;"
            />
            <div
              class="text-row"
              style="width: 8.75rem; height: 0.8125rem; background-color: rgb(230, 230, 230); margin-top: 0.625rem;"
            />
          </div>
        </div>
        <div
          class="ViewStyled-sc-rqe5sa-0 cEttAk"
          display="flex"
        >
          <div
            class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__StartBaseLineStyled-sc-okjczv-1 fZhgsY ifLECW"
            height="7"
            opacity="0.2"
            width="51"
          />
          <div
            class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__MiddleBaseLineStyled-sc-okjczv-2 bgjtXS dbuCnc"
            height="7"
            opacity="0.2"
            width="16"
          />
          <div
            class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__MiddleBaseLineStyled-sc-okjczv-2 iRkeGL dbIHPc"
            height="7"
            opacity="0.2"
            width="14"
          />
          <div
            class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__EndBaseLineStyled-sc-okjczv-3 cNBCGI fKiEZo"
            height="7"
            opacity="0.2"
            width="20"
          />
          <div
            class="LineStyled__BaseLineStyled-sc-okjczv-0 LineStyled__LineScoreStyled-sc-okjczv-4 ipAKRB hjZOmd"
            opacity="0.7"
            width="14"
          />
        </div>
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 ViewStyled__BackDropPointLocker-sc-rqe5sa-7 fTsvHV WdHgc"
      />
    </aside>
  </div>
  <div
    class="ViewStyled-sc-rqe5sa-0 PopupSuccessStyled__AnimatedPopupSuccess-sc-1umv97q-0 gKydeh bnNRjk"
    data-testid="popup-success"
    display="flex"
  >
    <div
      class="ViewStyled-sc-rqe5sa-0 kgTlIu"
      display="flex"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2730%27%20height=%2730%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <div
        class="ViewStyled-sc-rqe5sa-0 eaFPBc"
        display="flex"
      >
        <h3
          class="TextStyled-sc-9xia41-0 TitleStyled-sc-t58pjn-0 dZypbt dXQqcD"
          color="#FFF"
          font-family="AdobeCleanBold"
          font-size="24"
          letter-spacing="0.9"
          width="100%"
        >
          Success
        </h3>
        <p
          class="TextStyled-sc-9xia41-0 jJKZfr"
          color="#EAEAEB"
          font-size="13"
          font-weight="300"
          letter-spacing="0.9"
          width="100%"
        >
          Your bill was successfully linked.
        </p>
      </div>
    </div>
    <button
      class="ButtonStyled-sc-1ys9v52-0 bRMDWm"
      data-testid="button"
    >
      <span
        class="ButtonStyled__ButtonIconStyled-sc-1ys9v52-1 DjHyU"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 efewJe"
          display="inline-block"
          height="15"
          width="15"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
        </div>
      </span>
    </button>
  </div>
</div>
`;
